### DISCUSSION TITLE
Show resultList when clicking to input search

### DISCUSSION THREAD

**Discussion by svilder** [TarekRaafat/autoComplete.js#218] [Q&A] [answered]
When you have big list, and the user don't have a clear idea of what to look for, I would like to implement : 
When you click on the input, you show a result list with maybe the 15 first results with no query, so you can have an idea of what to look for, or a previously choosen suggestion list, even better (so you don't have to load the entire list). 

Have anyone any idea how I could do that ? 

Thanks 🙏

**Comment by TarekRaafat** [comment, marked as answer]
Hello @svilder,

Try the below `config` and let me know how it goes.

```js 
trigger: {
    event: ["input", "focus"],
    condition: () => true,
}
```

**Reply by paocom** [reply]
How about version 10+ ?

**Reply by TarekRaafat** [reply]
Hello @paocom,

Here you go, use this [code](https://tarekraafat.github.io/autoComplete.js/#/how-to-guides?id=open-list-on-focus) and add to it the same `condition` code above, and let me know how it goes.

Cheers, and have a nice day! :)

**Reply by asheerrizvi** [reply]
@TarekRaafat I did something like this for opening the list on focus, but it doesn't seem to work. I want to open the suggestions when the value length is zero.

![image](https://user-images.githubusercontent.com/17976252/124347348-e0d3ff80-dc01-11eb-83af-38afd975f8b4.png)

**Reply by TarekRaafat** [reply]
Hello @asheerrizvi,

You'll need to add to your config. `threshold: 0,` and it should work then.

Please try it and let me know how it goes.

**Reply by asheerrizvi** [reply]
@TarekRaafat oh silly me. Thanks a lot brother it is working fine now. Also, we are all thankful to you for creating, maintaining and continuing to contribute to this super cool package. 🙌

**Reply by TarekRaafat** [reply]
@asheerrizvi thanks a lot man, much appreciated, and I'm happy that it's useful for you.

Cheers, and have a nice day! :)
